internal/librarian/version.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package librarian
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/googleapis/librarian/internal/cli"
+)
+
+var CmdVersion = &cli.Command{
+	Name:  "version",
+	Short: "Print version information.",
+	Run:   runVersion,
+}
+
+func runVersion(ctx context.Context) error {
```

Comment:
Inline the function.

Reply:
I'd be interested in whether there are common conventions around this for Go. I can see that it's nice and short - but if every other command is using a pattern of a function called `runXyz`, is that consistency worth continuing?